### DISCUSSION TITLE
Failed to run OHIF plugin

### DISCUSSION THREAD

**Discussion by spandan-roy-ainostics** [Project-MONAI/MONAILabel#586] [Q&A]
I have installed monailabel and built the OHIF plugin as explained in the documentation. The monailabel server is running as well, however, when I try to go to http://localhost:8000/ohif/ , it returns a blank webpage. Can someone tell me where I am going wrong?

**Comment by SachidanandAlle** [comment]
You also need a dicom Web running somewhere for ohif.. the same you used as --studies

**Reply by spandan-roy-ainostics** [reply]
I have set up an Orthanc server and it's running as well.

**Comment by SachidanandAlle** [comment]
You can check browser logs.. also I hope you have added some images into Orthanc

**Reply by spandan-roy-ainostics** [reply]
So I checked the browser logs and it throws the following errors

<img width="1255" alt="image" src="https://user-images.githubusercontent.com/87641486/149789569-c63dcc5b-0a25-47eb-9ad5-1517e642a302.png">

It's weird because I can see those files are present in the static/ohif folder. Is there a way to validate if monailabel is connecting to the orthanc server properly?

**Reply by SachidanandAlle** [reply]
what is the monai label version you are using? how u have installed monailabel?

**Reply by SachidanandAlle** [reply]
Because the urls are not correct.. there should be /ohif  e.g. http://localhost:8000/ohif/app-config.js

**Reply by spandan-roy-ainostics** [reply]
I installed monailabel using pip, it's monailabel 0.1

<img width="578" alt="image" src="https://user-images.githubusercontent.com/87641486/149910450-29591a6e-053f-4eb7-bf46-54732dd5e42c.png">

**Reply by diazandr3s** [reply]
Hi @spandan-roy-ainostics, thanks for your reply. If you'd like to work with OHIF, please update MONAI Label version to the latest one or at least the version 0.2.0. Version 0.1 doesn't come with OHIF support.

Here is the release summary: https://github.com/Project-MONAI/MONAILabel/releases/tag/0.2.0

**Reply by spandan-roy-ainostics** [reply]
I tried installing the latest version using pip, but it didn't contain the OHIF plugin. I have tried cloning the source code and building the plugin myself but it says that there's no package.json.

**Reply by diazandr3s** [reply]
Which browser are you using? Some Firefox versions don't load the OHIF plugin. Could you please try Google Chrome?
From where did you get the no package.json warning/error?

**Comment by SachidanandAlle** [comment]
Are you still facing the issue?
